pkg/bucket/encryption/bucket-sse-config.go

```diff
@@ -0,0 +1,103 @@
+/*
+ * MinIO Cloud Storage, (C) 2020 MinIO, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package cmd
+
+import (
+	"encoding/xml"
+	"errors"
+	"io"
+)
+
+const (
+	// AES256 is used with SSE-S3
+	AES256 SSEAlgorithm = "AES256"
+	// AWSKms is used with SSE-KMS
+	AWSKms SSEAlgorithm = "aws:kms"
```

Comment:
these dont have to be defined - just use crypto.SSEAlgorithmAES256 and crypto.SSEAlgorithmKMS

Reply:
minio-go will be using this pkg for implementing bucket encryption SDK APIs. So, it makes sense to have all related constants in the same package.

Reply:
We shouldn't use server code in minio-go @krisis and we will never import them in this manner. 

This pkg is not publicly consuming we don't have to provide any API guarantees etc. 

minio-go needs to have it's own implementation it won't depend on server constants. 

Also @poornas redefining it here is not a problem because crypto might not have same requirements as this package. This package is self contained to parse AWS S3 spec details.